site/docs/relations/physical_relations.md

```diff
@@ -17,13 +17,15 @@ The hash equijoin join operator will build a hash table out of the right input b
-| Join Type           | One of the join types defined in the Join operator.          | Required                 |
+| Property            | Description                                                                                                                                                                                                                                                                                     | Required                                                                   |
+|---------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------|
+| Left Input          | A relational input.                                                                                                                                                                                                                                                                             | Required                                                                   |
+| Right Input         | A relational input.                                                                                                                                                                                                                                                                             | Required                                                                   |
+ | Left Keys           | A collection of fields reference in probe table                                                                                                                                                                                                                                                 | Optional, defaults to blank if during join predicate is specified          |
+ | Right Keys          | A collection of fields reference in build table                                                                                                                                                                                                                                                 | Optional, defaults to blank if during join predicate is specified          |             
+| During Join Predicate | A boolean condition that describes whether each record from the left set "match" the record from the right set. The condition must only include the following operations: AND, ==, field references, is not distinct from. Field references correspond to the direct output order of the data.  | Optional, defaults true if left keys and right keys are both specified.    |
```

Comment:
I don't see this in the proto definition

Reply:
Removed